{"name": "Проверка текста ошибки при попытке создания курьера с уже существующим логином", "status": "passed", "start": 1747565509326, "stop": 1747565509456, "uuid": "e401f713-0183-4dc9-acf8-2bbae537a864", "historyId": "b37c628ed87888dcad1b505c1d68ad90", "testCaseId": "b37c628ed87888dcad1b505c1d68ad90", "fullName": "tests.couriers.test_create_courier.TestCreateCourier#test_create_same_courier_error_message", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Pro-Mikhail.local"}, {"name": "thread", "value": "35443-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_create_courier"}]}